mobile/src/main/res/values/strings.xml

```diff
@@ -416,4 +416,10 @@
     <string name="item_update_widget_auto_generate_widget_label">Auto generate widget label</string>
     <string name="item_update_widget_widget_label">Widget label</string>
     <string name="item_update_widget_default_label"><![CDATA[<Item label>: <Action>]]></string>
+    <string name="data_saver_title">Data saver</string>
+    <string name="data_saver_on">Reduce data usage on mobile data</string>
```

Comment:
Actually the data usage is reduced on metered networks, but I'm not sure if every user knows what that means.